src/System.Security.Cryptography.Pkcs/tests/Pkcs9AttributeTests.cs

```diff
@@ -27,6 +27,42 @@ public static void Pkcs9AttributeAsnEncodedDataCtorNullOid()
             Assert.Throws<ArgumentNullException>(() => ign = new Pkcs9AttributeObject(a));
         }
 
+        [Fact]
+        public static void TestDateBefore1950()
+        {
+            DateTime dt = new DateTime(1949, 12, 31, 11, 59, 59);
+            try
+            {
+                var st = new Pkcs9SigningTime(dt);
+            } catch (Exception ex)
+            {
+                if (ex is ArgumentOutOfRangeException || ex is CryptographicException)
```

Comment:
Having the exceptions be normalized seems virtuous.  I'd wrap the ArgumentException inside the PAL WriteUtcTime, so that both PAL methods succeed or throw CryptographicException.

Then this test becomes

```C#
Assert.ThrowsAny<CryptographicException>(() => new Pkcs9SigningTime(dt));
```